doc/observability/tracing.md

```diff
@@ -0,0 +1,125 @@
+  namespace: istio-system
+  values:
+    meshConfig:
+      defaultConfig:
+        tracing: {}
+      enableTracing: true
+      extensionProviders:
+      - name: jaeger-tempo
+        opentelemetry:
+          port: 4317
+          service: tempo.tempo.svc.cluster.local
+```
+
+### Kuadrant Tracing Configuration
+
+The Authorino and Limitador components have request tracing capabilities.
+Here is an example configuration to enable and send traces to a central collector.
+Ensure the collector is the same one that Istio is sending traces so that they can be correlated later.
+
+```yaml
+apiVersion: kuadrant.io/v1beta2
+kind: AuthPolicy
+metadata:
+  name: authpolicy
+  namespace: default
+spec:
+  rules:
+    authentication: {}
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: mygateway
+  tracing:
+    endpoint: rpc://tempo.tempo.svc.cluster.local:4317
+    insecure: true
+---
+apiVersion: kuadrant.io/v1beta2
+kind: RateLimitPolicy
+metadata:
+  name: limitpolicy
+  namespace: default
+spec:
+  limits: {}
+  targetRef:
+    group: gateway.networking.k8s.io
+    kind: HTTPRoute
+    name: mygateway
+  tracing:
+    endpoint: rpc://tempo.tempo.svc.cluster.local:4317
+    insecure: true
+```
+
+Once the changes are applied, the authorino and limitador components will be redeployed tracing enabled.
+
+**Note:**
+
+There are [plans](https://github.com/Kuadrant/architecture/issues/48) to consolidate the tracing configuration to a single location i.e. the Kuadrant CR.
+This will eventually eliminate the need to configure tracing in both the Authorino and Limitador CRs.
+
+**Important:**
+
+Currently, trace IDs [do not propagate](https://github.com/envoyproxy/envoy/issues/22028) to wasm modules in Istio/Envoy, affecting trace continuity in Limitador.
+This means that requests passed to limitador will not have the relavant 'parent' trace ID in its trace information.
+If however the trace initiation point is outside of Envoy/Istio, the 'parent' trace ID will be available to limitador and included in traces passed to the collector.
+This has an impact on correlating traces from limitador with traces from authorino, the gateway and any other components in the path of requests.
+
+## Troubleshooting Flow Using Traces and Logs
+
+Using a tracing interface like the Jaeger UI or Grafana, you can search for trace information by the trace ID.
+You may get the trace ID from logs, or from a header in a sample request you want to troubleshoot.
+You can also search for recent traces, filtering by the service you want to focus on.
+
+Here is an example trace in the Grafana UI showing the total request time from the gateway (Istio), the time to check the curent rate limit count (and update it) in limitador and the time to check auth in Authorino:
+
+<img src="./grafana_trace.png" alt="Trace in Grafana UI" width="800"/>
+
+In limitador, it is possible to enable request logging with trace IDs to get more information on requests.
```

Comment:
Perhaps it's worth mentioning how here? This requires the log level to be increased to at least debug, so the verbosity must be set to 3 or higher in the limitador CR:

```sh
kubectl patch -n kuadrant-system limitador/limitador --type=merge -p '{"spec":{"verbosity":3}}'
```